### PR TITLE
Bugfixes

### PR DESCRIPTION
**Pull request by hcarneiro**

### CODE REVIEW

File: js/build.js [hunk 1]
```diff
@@ -129,12 +129,19 @@ if (typeof jQuery !== 'undefined') {
...
+    $().tooltip('destroy');
+    $(this).parents('.chats').find('.chat-text[aria-describedby]').tooltip('hide');
     $(this).tooltip('toggle');
   });
 
   $(document).on('click', '.tooltip', function() {
-	   copiedElem.copyText();
-    $(this).tooltip('hide');
+    var _this = $(this);
+	  copiedElem.copyText();
```
Comment (squallstar) [inline js/build.js]: indentation; also, rename `_this` to `$el`

File: js/build.js [hunk 2]
```diff
@@ -130,12 +130,18 @@ if (typeof jQuery !== 'undefined') {
...
+    $(this).parents('.chats').find('.chat-text[aria-describedby]').tooltip('hide');
     $(this).tooltip('toggle');
   });
 
   $(document).on('click', '.tooltip', function() {
-	   copiedElem.copyText();
-    $(this).tooltip('hide');
+    var $el = $(this);
+    copiedElem.copyText();
+    $(this).find('.tooltip-inner').text('Copied!');
```
Comment (squallstar) [inline js/build.js]: `$el`

File: js/build.js [hunk 3]
```diff
@@ -191,25 +197,23 @@ if (typeof jQuery !== 'undefined') {
     $(this).addClass('sending');
     holder.addClass('sending');
 
-    $message.val('');
```
Comment (squallstar) [inline js/build.js]: watch out, because this means if I press twice the button it will send the message twice

Comment (hcarneiro) [inline js/build.js]: You can't click the button twice because I removed the pointer events with CSS when `sending`

**Pull request merged**